src/templates/pt/2019/base.html

```diff
@@ -0,0 +1,229 @@
+{% extends "base/2019/base.html" %}
+
+{% block description %}Web Almanac é um estado anual do relatório da Web que combina a experiência da comunidade da Web
+com os dados e as tendências do arquivo HTTP.{% endblock %}
```

Comment:
Combine these to keep line numbering same as English:

```suggestion
{% block description %}Web Almanac é um estado anual do relatório da Web que combina a experiência da comunidade da Web com os dados e as tendências do arquivo HTTP.{% endblock %}
```

Reply:
strange that, for me it was line by line, I believe it is the beautify plugin.
Thanks for reminding me @bazzadp